sdk/storage/azure-storage-file-share/azure/storage/fileshare/_share_client.py

```diff
@@ -314,6 +316,13 @@ def create_share(self, **kwargs):
 
         :keyword int timeout:
             The timeout parameter is expressed in seconds.
+        :keyword protocols:
+            Protocols to enable on the share. Only one protocol can be enabled on the share.
+        :paramtype protocols: str or ~azure.storage.fileshare.ShareProtocols
```

Comment:
type should be a list?

Reply:
No it shouldn't, as discussed with @annatisch, we will opt for using a string since users are allowed to use only one protocol at the moment. It doesn't make sense to have them pass ['SMB'] or ['NFS']. As Anna recommended we will stick to string for now and then in the future we can simply change the type to `Union[str, [protocols]]`. However, it is important to note that we are in fact deserializing the enabled protocols on a share into a list, i.e in `ShareProperties` (as discussed in the archboard meeting)

Reply:
as long as you have a plan to make it a list in the future I'm good with it!